CHANGELOG.md

```diff
@@ -1,5 +1,113 @@
 # Changelog
 
+## v0.17.0 (2017-8-2)
+
+### Enhancements
+
+#### :gear: Testing Electron applications ([testcafe-browser-provider-electron](https://github.com/DevExpress/testcafe-browser-provider-electron))
+
+We have prepared a browser provider that allows you to test Electron applications with TestCafe.
+
+Getting it to work is simple. First, install the plugin from npm.
+
+```sh
+npm install testcafe-browser-provider-electron --save-dev
+```
+
+Then create a `.testcafe-electron-rc` file that contains configurations for the Electron plugin.
+The only required setting here is `mainWindowUrl` - a URL or path to the main window page relative to the application directory.
```

Comment:
Let's make sentences shorter.

The only required setting here is `mainWindowUrl`. It's a URL (or path) to the main window page relative to the application directory.